bom/pom.xml

```diff
@@ -87,6 +87,8 @@ quarkus.micrometer.binder.kafka.enabled=false
 quarkus.micrometer.binder.system=true
 quarkus.micrometer.binder.vertx.enabled=true
 quarkus.micrometer.export.prometheus.path=/prometheus
+# Hono custom Sampler shall not be wrapped by parent-based Sampler
+quarkus.opentelemetry.tracer.sampler.parent-based=false
```

Comment:
The code in *SamplerProducer* seems to suggest that the value of this property is being ignored and we always create a parent based sampler. Or am I mistaken?

Reply:
Good point. I've adapted the corresponding code in `SamplerProducer`.
With the new Quarkus version, the `parent-based` property is handled differently. If now set to `true`, it means that the custom sampler will be wrapped with a parent-based Sampler, meaning the parent-based Sampler overrides the custom Sampler when dealing with child spans. We don't want that here.